client/src/components/dashboard/Dashboard.js

```diff
@@ -1,16 +1,75 @@
-import Container from 'react-bootstrap/Container';
+/**
+ * This file exports the content in of Researchify Dashboard Page
+ */
+import React, { useState } from 'react';
+import { Container, CardGroup, Card, Button } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+// Redux
 import { useSelector } from 'react-redux';
+// icons
+import { BsPencilSquare, BsServer, BsDisplayFill } from 'react-icons/bs';
+// css
 import './Dashboard.css';
+// Component
+import TemplateSelector from './TemplateSelector';
 
+/**
+ * Dashboard Component
+ */
 const Dashboard = () => {
-  const userName = useSelector((state) => state.team?.teamName);
+  const teamId = useSelector((state) => state.team.teamId);
+
+  // Display pop up window
+  const [displayModal, setDisplay] = useState(false);
+  const showModal = () => setDisplay(true);
+  const closeModal = () => setDisplay(false);
 
   return (
-    <main>
-      <Container fluid>
-        You are currently viewing dashboard page. Team name: {userName}
-      </Container>
-    </main>
+    <Container fluid className="researchify-dashboard-container">
+      <Card className="text-center researchify-dashboard-card">
+        <Card.Body>
+          <Button onClick={showModal}> Select a theme </Button>
+        </Card.Body>
+
+        <Card.Body className="researchify-dashboard-card-description">
+          Click the button to select or update your website theme.
+        </Card.Body>
+
+        {/* Bottom layer of the card with three icons */}
+        <CardGroup className="researchify-dashboard-card-group">
+          <Card>
+            <Link className="researchify-dashboard-card-link">
+              <Card.Body>
+                <BsPencilSquare className="researchify-dashboard-card-icons" />
+              </Card.Body>
+              <p>Editor</p>
+            </Link>
+          </Card>
+          <Card>
+            <Link className="researchify-dashboard-card-link">
+              <Card.Body>
+                <BsServer className="researchify-dashboard-card-icons" />
+              </Card.Body>
+              <p>API Acess Manager</p>

```

Comment:
What's this page going to be for?

Reply:
So before it was to direct users to a page where they can manage their tokens and stuff, like twitter token or github. But since we're not getting their GitHub token anymore, I just can't decide what to replace it and keep the icon for now.

Reply:
Yeah, it can probably be removed because I can't think what else it'd be for either

Reply:
Removed, it looks kinda weird when there're only 2 icons under tho XD